docs/configuration.md

```diff
@@ -167,3 +167,36 @@ Any flag that has a value that is a comma separated list on the command line bec
     [profile]
     cpu-time = "30s"
     ```
+##### Metric Service
+* Description: which stats service to use (StatsD or ExpVar)
+* Flag: `--metric.service=statsd`
+* Env: `PILOSA_METRIC.SERVICE=statsd'
+* Config:
+
+    ```toml
+    [metric]
+    service = “statsd”
+    ```
+
+##### Metric Host
+* Description: where is the StatsD service hosted
```

Comment:
this sounds like a question. 
can this just be `Address of the StatsD service host.`